docs/maps/map-collection.md

```diff
@@ -20,195 +20,83 @@ In the [map collection reference](http://code.highcharts.com/mapdata/), each map
 1. Add the map as a JavaScript element:
 
 ```html
-<script src="[http://code.highcharts.com/mapdata/custom/world.js](http://code.highcharts.com/mapdata/custom/world.js)"></script>
+<script src="http://code.highcharts.com/mapdata/custom/world.js"></script>
 ```
 
-You can alternatively link to a specific version or subversion of the map at http://code.highcharts.com/mapdata/**1.1**/custom/world.js. 
+<p>You can alternatively link to a specific version or subversion of the map at <code>http://code.highcharts.com/mapdata/<strong>1.1</strong>/custom/world.js</code>.</p>
```

Comment:
Should be `https://...`. Try doing a general search and replace in all the `/docs` for insecure URLs to our own services (`http://code.highcharts.com`, `http://api.highcharts.com`).